mirage-runtime.opam

```
@@ -26,7 +26,10 @@ depends: [
   "lwt" {>= "4.0.0"}
   "alcotest" {with-test}
 ]
-conflicts: [ "result" {< "1.5"} ]
+conflicts: [
+  "result" {< "1.5"}
+  "ppxlib" {>= "0.29.0"} #0.29 provides a vendored ppx_sexp_conv
```

```suggestion
  "ppxlib" {!= "0.29.0"} #0.29.0 provides a vendored ppx_sexp_conv
```
https://github.com/ocaml-ppx/ppxlib/pull/386 just got merged and 0.29.1 should get released soon.